docs/internal/config/Template-Builder.md

```diff
@@ -1,15 +1,140 @@
+<!-- tocstop -->
+## Overview
+
+One of the simplest ways to customize the system is to alter what information is presented to an applicant on an application form. 
+Another simple way is to alter what data the applicant must provide when filling out an application form. 
+Methods to do each of these are explained below.
+
+### How to edit a template
+
+To find the Template Builder (visible only to Admin users), go to: Configurations > Templates/Procedures
+![Find template builder tool](../images/Template-builder1.png)
+
+Select the edit button for the Form Template you wish to edit. The names of the templates are highlighted in pink:
+![Edit template](../images/Template-builder2.png)
+ 
+**Note**: 
+- Form Templates can only be altered if there are no existing applications for this template version. 
+- If applications exist for this version then first you must duplicate and then the copy of this Template can then be altered. 
+   1. To change a copy of the current template, click once on the row to see all versions
+   2. Then click on the edit icon on the last version (In Draft) to edit!
 
-TODO
 ## General tab
 
+When you open one Template to view and edit the landing page is the General. As you can see below following our example of editing "Add user to company" template.
+
+**Note**: Form Templates must be in ‘DRAFT’ status before they can be edited. 
+Click the ‘Make Draft’ button in the config builder
+![General tab](../images/Template-builder3.png)
+
+In this tab is possible to change the status of this Template **version** to Make Available, Make Draft or Disable the application (for all end-users).
 ### Displayed options
 
-#### canApplicantMakeChangess
+| option             | template_column            | optional | default    | description                                                                                                                    |
+| ------------------ | -------------------------- | -------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| Name               | name                       | no       | -          | Name of a **single application** displayed to users (in different areas of the App)                                            |
+| Name plural        | name_plural                | no       | -          | Name of the **group of applications** displayed to users (in different areas of the App)                                       |
+| Code               | code                       | no       | -          | Reference to application type. In case a new code is created this template version will be displayd as a new template type!    |
+| Serial Pattern     | serial_patern              | yes      | **?**      | The pattern to be used when generating a new application - used regular expression                                             |
```

Comment:
Typo: serial_patern

Also, I'm sure I mentioned this in the last review, but seems to be gone now...

Instead of "- used regular expression" it should say something like:
```
uses "Generate Text String" action (with link to https://github.com/openmsupply/conforma-server/wiki/List-of-Action-plugins#generate-text-string)

Reply:
Thanks! I'll add this link